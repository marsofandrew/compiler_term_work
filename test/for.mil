BEGIN

    FOR i := READ; i < 8; i := i + 1 DO
        WRITE(i)
    OD

END