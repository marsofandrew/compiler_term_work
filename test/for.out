0:	INPUT
1:	STORE	0
2:	LOAD	0
3:	PUSH	8
4:	COMPARE	2
5:	JUMP_NO	23
6:	LOAD	0
7:	PUSH	1
8:	ADD
9:	LOAD	0
10:	PRINT
11:	LOAD	0
12:	PUSH	2
13:	MULT
14:	PUSH	8
15:	COMPARE	3
16:	JUMP_NO	21
17:	LOAD	0
18:	PUSH	2
19:	MULT
20:	PRINT
21:	STORE	0
22:	JUMP	2
23:	STOP
